lib/help.js

```diff
@@ -11,6 +11,7 @@ module.exports = `
       --timeline, -i     Display timeline view
       --delete, -d       Delete item
       --check, -c        Check/uncheck task
+      --begin, -b        Start/stop task
```

Comment:
Instead of `stop`, we could maybe go with something a bit more fitting like `pause`, for example:

```
      --begin, -b        Start/pause task
```